params/config.go

```diff
@@ -486,7 +486,8 @@ type BorConfig struct {
 	StateReceiverContract    string                 `json:"stateReceiverContract"`    // State receiver contract
 	OverrideStateSyncRecords map[string]int         `json:"overrideStateSyncRecords"` // override state records count
 	BlockAlloc               map[string]interface{} `json:"blockAlloc"`
-	BurntContract            map[string]string      `json:"burntContract"` // governance contract where the token will be sent to and burnt in london fork
+	BurntContract            map[string]string      `json:"burntContract"`         // governance contract where the token will be sent to and burnt in london fork
+	JaipurBlock              int                    `json:"jaipurBlock,omitempty"` // Jaipur switch block (nil = no fork, 0 = already on jaipur)
```

Comment:
Shouldn't we use it outside bor config? like London fork flag

Reply:
we tried that but there are few places where we wont have whole config
in snapshot.go basically

Reply:
Can we convert it to `big.Int` similar to other fork block numbers?